### DISCUSSION TITLE
Avoid implausible values

### DISCUSSION THREAD

**Discussion by markusdd** [arendst/Tasmota#20969] [Q&A]
Hi,

maybe this has even been asked before and I'm not able to find it, but I have an annoying issue with my power meter.

This is being read out out via the classic infrared head which then send the values to the ESP.

This works very well, but once every few weeks I get a 0-reading on the consumed kWh or supplied kWh, which of course messes up the database and all visualizations because suddenly you have a consumption of Megawatthours in a single day.

Is there a way to filter implausible values? Either on Tasmotas end, in the SML script or when receiving them into influxdb via home assistant?

0 is never a sensible value for aggregated consumption or supply, so I'd like to drop them.

**Comment by gemu2015** [comment]
try enabling median filter for that meter
example:   (the value 16 in meter flags enables it, see docs)
 +1,3,s,16,9600,SML1

**Reply by markusdd** [reply]
good idea! How does this differ to counter with pullup? I cannot find any documentation what counter with pullup means.
